reactioncommerce/reaction
# Instagram Login/Registration

This resolves #2559

### Steps to test
1. Register an app with Instagram at http://instagram.com/developer/clients/register
1. Fill out the given form but make sure that redirect url as shown as follows-
OAuth redirect_uri: `<your-server-domain>:<port>/_oauth/instagram` For example `http://localhost:3000/_oauth/instagram`
1. After registration, copy the `clientId` and `client secret`.
1. Signin to Reaction as admin
1. On the dashboard, under Login Services, toggle on Instagram Sign In
1. On a new browser, click on the authentication dropdown in nav and signin with instagram.
